flink-runtime-web/web-dashboard/app/partials/jobs/job.jade

```diff
@@ -43,6 +43,10 @@ nav.navbar.navbar-default.navbar-fixed-top.navbar-main(ng-if="job")
     span.navbar-info-button.btn.btn-default(ng-click="cancelJob($event)")
       | Cancel
 
+  .navbar-info.last.first(ng-if!="job.type=='STREAMING' && (job.state=='RUNNING' || job.state=='CREATED')")
```

Comment:
Yes. Quite odd indeed. Can you point out the resource where you saw this? I'm a little curious.